Source/WebCore/inspector/agents/InspectorTimelineAgent.cpp

```diff
@@ -860,4 +842,23 @@ void InspectorTimelineAgent::pushCurrentRecord(Ref<JSON::Object>&& data, Timelin
     pushCurrentRecord(createRecordEntry(WTFMove(data), type, captureCallStack, frame, startTime));
 }
 
+void InspectorTimelineAgent::didCompleteRenderingFrame()
+{
+#if PLATFORM(COCOA)
```

Comment:
Is this all really Cocoa-specific?

Reply:
Yeah this code is only used by the runloop codepath which is Cocoa specific.